theme/secondary-menu/_base.scss

```diff
@@ -7,8 +7,9 @@
 }
 
 #wb-sec {
-	margin-top: 20px;
```

Comment:
Not having any background color behind _Content page_ could be a issue with people with poor vision.